changelogs/fragments/change_import_resoure_module.yaml

```diff
@@ -0,0 +1,3 @@
+---
+minor_changes:
```

Comment:
Since this is just a test change and doesn't affect end user experience, can we make this a `trivial` change?